users/viam/0_keyboard_post_init_user.c

```diff
@@ -0,0 +1,8 @@
+#if defined(BETTER_DRAGSCROLL)
+    #include "better_dragscroll.h"
+#endif // defined(BETTER_DRAGSCROLL)
+#if ( defined(BETTER_DRAGSCROLL) && ((defined(BETTER_DRAGSCROLL_SCRLK_ENABLE)) || defined(BETTER_DRAGSCROLL_CAPLK_ENABLE) || defined(BETTER_DRAGSCROLL_NUMLK_ENABLE) || (defined(VIA_ENABLE) && defined(PLOOPY_VIAMENUS))) || defined(COMMUNITY_MODULE_BETTER_DRAGSCROLL_ENABLED))
+    void keyboard_post_init_user(void) {
+      led_initialize_better_dragscroll();
+    }
```

Comment:
@plodah despite the rest of this PR content, a similar change to this should still be done because at the current state of this repository, if you plug the trackball when a drag-led is active, then dragging isn't activated by default. But it always starts in normal mode until yo toggle it. And that's inconsistent.